partner_credit_limit_history/models/res_partner.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    credit_history_ids = fields.One2many(
+        comodel_name="res.partner.credit.history",
+        inverse_name="partner_id",
+        string="Credit History",
+        copy=False,
+    )
+
+    def action_partner_credit_limit_wizard(self):
+        action = self.env.ref(
```

Comment:
You can do this with an action in the button for not having Python code for such simple action.